ckanext/cloudstorage/fanstatic/scripts/cloudstorage-multipart-upload.js

```diff
@@ -241,13 +242,24 @@ ckan.module('cloudstorage-multipart-upload', function($, _) {
                 return;
             }
             event.preventDefault();
-
-            try{
-                this._onDisableSave(true);
-                this._onSaveForm();
-            } catch(error){
-                console.log(error);
+            var dataset_id = this.options.packageId;
+            this._btnClick = $(event.target).attr('value');
```

Comment:
maybe, `_clickedBtn` or something, that hints about content of this variable. Now it looks like you are storing here some callback